lib/consensus/ballot_send_record.go

```diff
@@ -24,6 +24,8 @@ func NewBallotSendRecord(nodeAlias string) *BallotSendRecord {
 	return p
 }
 
+// SetSent sets that the ballot of this ISAACState has already been sent.
+// This is to prevent one node from retransmitting another result.
```

Comment:
Big big 👍 for documenting what the function is for. It's much more important than what the function do (which one can understand from the code).